hack/support-bundle-collector.sh

```diff
@@ -28,6 +28,7 @@ else
     echo "No OS collector found"
 fi
 
+"collector-default" $HOST_PATH $BUNDLE_DIR
```

Comment:
What is the reason for double quoting the `collector-default`?

Reply:
Sorry, this is a typo on my part. I have fixed it.